src/Game.cpp

```diff
@@ -87,8 +87,9 @@ int EnterLoadingScreen()
     uint32_t index = 0;
 
     /* Calculater the position */
-    int px = (GetWidth() / 2) - (1024 / 2);
-    int py = (GetHeight() / 2) - (768 / 2);
+    Size loadingBgSz = { 1024, 768 };

```

Comment:
encapsulate this in a static class, see other fields below..
- bar size
- resolution

Reply:
Could you please show example?
I just start look at source code

Reply:
In the file GameManger.cpp there are global variables defined for resolution ect., however acceptable resolutions are stored in game.cpp.
This shouldn't be seperate.
Additionally the GameManager should be encapsuleted in a class.

Reply:
For drawing UI on the screen, I think there should be a function like drawOnScreen(posx, posy, Size), instead of writing the same code all over.